### ISSUE TITLE
put project on PyPI

### ISSUE THREAD

**Issue by graingert** [matiasb/python-unidiff#33]
#10 was never completed, the project is not installable with pip

you should do:

```
python3 -m venv /tmp/unidff
source /tmp/unidiff/bin/activate
pip install -U pip setuptools wheel
python setup.py sdist bdist_wheel upload
```

**Comment by matiasb** [comment]
FWIW, v0.5.2 is pip-installable (pip install unidiff) and it is available in PyPI:
https://pypi.python.org/pypi/unidiff

I will be uploading next version, hopefully soon, as a wheel too (thanks!).

**Comment by graingert** [comment]
Sweet, thanks very much

**Comment by graingert** [comment]
@matiasb there doesn't seem to be any files available

**Comment by graingert** [comment]
It seems the unidiff project is missing a https://pypi.org/project/unidiff/#files section.

Can you reupload with:

```
python3 -m venv /tmp/unidff
source /tmp/unidiff/bin/activate
pip install -U pip setuptools wheel
python setup.py sdist bdist_wheel upload
```

**Comment by matiasb** [comment]
Ok, there was something weird there... Re-uploaded and released as version 0.5.3.
Sorry for the issue, will make sure to review/update metadata during the next days.
Let me know if you notice any other problem. Thanks a lot for reporting!

**Comment by graingert** [comment]
Thanks!
